Integrations/ZeroFox/ZeroFox.py

```diff
@@ -0,0 +1,510 @@
+from distutils.util import strtobool
+from typing import Dict
+
+import demistomock as demisto
```

Comment:
add `from CommonServerUserPython import *`
make sure these are the first three lines